code/mathematical_algorithms/src/karatsuba_multiplication/karatsuba_multiplication.cpp

```diff
@@ -0,0 +1,88 @@
+#include<iostream>
+using namespace std;
+
+// To pad both input strings with 0's so they have same size
+int equalLength(string &bitStr1, string &bitStr2)
+{
+	int strLen1 = bitStr1.size();
+	int strLen2 = bitStr2.size();
+
+	if(strLen1>strLen2){
+		int numZerosToPad = strLen1 - strLen2;
+		for(int i=0; i< numZerosToPad; i++)
+			bitStr2 = '0' + bitStr2;
+	}
+
+	else if(strLen2>strLen1){
+		int numZerosToPad = strLen2 - strLen1;
+		for(int i=0; i< numZerosToPad; i++)
```

Comment:
Please spaces after/before all operators. Please spaces after keywords like `if`, `for`, etc.